### PR TITLE
[MIG] migrate business_requirement_etherpad from 10.0 to 12.0

### PR DESCRIPTION
**Pull request by ruter-lyu**

### CODE REVIEW

**Review by elicoidal — COMMENTED** [review]
LGTM
Please update the README

**Review by JoJoJoJoJoJoJo — APPROVED** [review]
LGTM

File: business_requirement_etherpad/README.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```
Comment (nikul-serpentcs) [inline business_requirement_etherpad/README.rst:3]: @ruter-lyu LGPL-3

File: business_requirement_etherpad/README.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+   :width: 600 px
+   :alt: Business Requirement List view
+
+
+Usage
+=====
+
+.. figure:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/222/10.0
```
Comment (nikul-serpentcs) [inline business_requirement_etherpad/README.rst:22]: 10 --> 12

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
netpicking

**Comment by pedrobaeza** [comment]
Closing as old and not green

**Pull request closed without merging**